index.html

```diff
@@ -61,6 +61,172 @@ <h2 class="body-text-header">I'm a Software Developer</h2>
 </svg>
 </div>
       </section>
+      <section class="project">
+        <div class="project-content">
+          <h3 class="underlined">My Recent Works</h3>
+        <div class="project-list">
+          <div class="project-item">
+            <div class="project-item-img">
+          <img src="/img/tonic2.png" alt="" class="img">
+        </div>
+      </div>
+      <div class="project-disc">
+      <div class="project-heading">Multi-Post Stories</div>
+      <p class="project-disc-main">A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standar dummy text.</p>
+
+      <div class="project-btns">
+        <button class="project-btns-list">css</button>
+        <button class="project-btns-list">html</button>
+        <button class="project-btns-list" id="bts">bootstrap</button>
+        <button class="project-btns-list">Ruby</button>
+      </div>
```

Comment:
- [ ] I am not sure if you actually have a plan to make this part of your code interactive with users but going by Figma design, it is just a list of the skills and stacks you used on a specific project. As a convention and best practice, it's best to use a `ul` and `li` to handle this section. Kindly list your skills with an `unordered list` :pray:. In fact, it will make your styling very easy :wink: